README.md

```diff
@@ -110,44 +110,48 @@ yarn add devicon
+<i class="devicon-devicon-plain-wordmark"></i>
 
-<!--  for git plain version colored with git main color (devicon-color.css or devicon.min.css required) -->
-<i class="devicon-git-plain colored"></i>
+<!--  for devicon plain version colored with devicon main color -->
+<i class="devicon-devicon-plain colored"></i>
 
-<!--  for git plain version with wordmark colored with git main color (devicon-color.css or devicon.min.css required) -->
-<i class="devicon-git-plain-wordmark colored"></i>
+<!--  for devicon plain version with wordmark colored with devicon main color -->
+<i class="devicon-devicon-plain-wordmark colored"></i>
 ```
 
 <p>
     An alternate way to use <code>devicon</code> is by copy/paste the raw svg code
     to your project.
 </p>
-<h4 id="getting-started-svg">Copy/paste svg code (from the <a href="https://github.com/devicons/devicon/tree/master/icons">svg folder</a> or the <a href="https://devicon.dev">project page</a></h4>
+<h4 id="getting-started-svg">Copy/paste SVG code (from the <a href="https://github.com/devicons/devicon/tree/master/icons">svg folder</a> or the <a href="https://devicon.dev">project page</a>)</h4>
 
 ```html
-<!--  for git plain version -->
-<svg class="devicon-git-plain" viewBox="0 0 128 128">
-  <path fill="#F34F29" d="M124.742,58.378L69.625,3.264c-3.172-3.174-8.32-3.174-11.497,0L46.685,14.71l14.518,14.518c3.375-1.139,7.243-0.375,9.932,2.314c2.703,2.706,3.462,6.607,2.293,9.993L87.42,55.529c3.385-1.167,7.292-0.413,9.994,2.295c3.78,3.777,3.78,9.9,0,13.679c-3.78,3.78-9.901,3.78-13.683,0c-2.842-2.844-3.545-7.019-2.105-10.521L68.578,47.933l-0.002,34.341c0.922,0.455,1.791,1.063,2.559,1.828c3.779,3.777,3.779,9.898,0,13.683c-3.779,3.777-9.904,3.777-13.679,0c-3.778-3.784-4.088-9.905-0.311-13.683C58.079,83.169,59,82.464,60,81.992V47.333c-1-0.472-1.92-1.172-2.856-2.111c-2.861-2.86-3.396-7.06-1.928-10.576L40.983,20.333L3.229,58.123c-3.175,3.177-3.155,8.325,0.02,11.5l55.126,55.114c3.173,3.174,8.325,3.174,11.503,0l54.86-54.858C127.913,66.703,127.916,61.552,124.742,58.378z"/>
-</svg>
+<!--  for devicon plain version -->
+<svg id="Devicon" class='devicon-devicon-plain' xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path id="plain" fill="#60be86" d="M64,7.83H4.77L14.95,95.13l49,25,.06,0,49.07-25L123.23,7.83Zm42.77,54.86c0,.88,0,1.67-.77,2L73.25,80.44l-2.42,1.13-.27-3.15V72.23l.24-1.57,1.09-.47L95.07,59.81l-21.54-9.6L64.35,68.34,58.9,78.87l-1.22,2.27-2.05-.9L22,64.71a2.42,2.42,0,0,1-1.45-2V56.91a2.39,2.39,0,0,1,1.42-2l34-15.73,3.21-1.44v9.66l.24,1.34-1.56.7L34.45,59.79,56.3,69.42l8.05-16,6.21-12.65,1.13-2.28,1.81.91L106,54.89c.73.35.76,1.14.76,2Z"/></svg>
 ```
 
 Add css rules in your stylesheet
 ```css
-.devicon-git-plain {
+.devicon-devicon-plain {
   max-width: 2em;
 }
 
 /* if you want to change the original color */
-.devicon-git-plain path {
+.devicon-devicon-plain path {
   fill: #4691f6;
 }
 ```
 
+<h4>You can also use the <code>img</code> tag and referencing an SVG directly from the repo.</h4>
```

Comment:
```suggestion
<h4>You can also use the <code>img</code> tag and referencing an SVG directly from the repository.</h4>
```

Reply:
I think this one does not need to be changed. We don't have to be 100% formal and repo is a reasonable shorthand (like develop for development branch)